### PR TITLE
MC-13608: Added documentation for the create site

### PR DESCRIPTION
**Pull request by fyrnaga**
#### Changes made

- Added the create site api doc

#### Related PRs
- PR # https://github.com/cloudops/cloudmc-stackpath-plugin/pull/140

### CODE REVIEW

File: source/includes/stackpath/_sites.md [hunk 1]
```diff
@@ -109,6 +109,75 @@ Attributes | &nbsp;
...
+{
+	"domain": "www.rpgfan.com",
+	"services": [
+		"CDN",
+		"SERVERLESS_EDGE_ENGINE",
+		"WAF"
+	],
+	"protocol": "HTTPS",
+	"hostname": "199.250.204.212",
+    "authMethod": "BASIC",
```
Comment (evanluc) [inline source/includes/stackpath/_sites.md]: Formatting

Comment (fyrnaga) [reply source/includes/stackpath/_sites.md]: e7a63e7dbb737d508bcde1e0f1fdf22ded45514f

**Review by evanluc — APPROVED** [review]

File: source/includes/stackpath/_sites.md [hunk 2]
```diff
@@ -109,6 +109,75 @@ Attributes | &nbsp;
...
+------- | -----------
+`domain`<br/>*string* | The domain name that will be used for the site.
+`services`<br/>*Array[string]* | Services list that will be used on the site. Possibles values are `CDN`,`SERVERLESS_EDGE_ENGINE` or  `WAF`.
+`protocol`<br/>*string* | Protocol that will be used to communicate with the hostname. Possibles values are `HTTP` or `HTTPS`.
+`hostname`<br/>*string* | The hostname to that will be used to get the information from. The hostname can be an IP or a name. It may include a specific port and a precise path as well. (e.g. 199.250.204.212:80/test) 
+ 
+ Optional | &nbsp;
+ ------- | -----------
+`authMethod`<br/>*string* | The authentication method to communicate with the hostname. Possibles values are `NONE` or `BASIC`. If not provided, it will default to `NONE` unless the username or password is provided. It would then default to `BASIC`.
+`username`<br/>*Array[Object]* | The username for the basic authentication. Required if authMethod is `BASIC` or if the password id provided
```
Comment (patrickspensieri) [inline source/includes/stackpath/_sites.md]: Just missing some periods at the end of `username` and `password` lines

**Review by patrickspensieri — APPROVED** [review]

File: source/includes/stackpath/_sites.md (hunk 2 above)
Comment (fyrnaga) [reply source/includes/stackpath/_sites.md]: 95e716efa6362b353a7fbad01bbcacf3407dd205

**Pull request merged**